packages/theming/default-theme/src/defaultColors.ts

```diff
@@ -138,6 +139,7 @@ export function getStockWebPalette(): ThemeColorDefinition {
       accent: globalTokens.color.brand.primary,
       blackTranslucent40: 'rgba(0,0,0,.4)',
     }),
+    ...createAliasTokens('light'),
```

Comment:
I like that we could add this on to the defaultColors =) I wonder if we could remove the `paletteFromFabricColors` call above in the future if nothing _actually_ depends on those colors.

Reply:
Maybe in the distant future? But it's possible clients use those color entries on a theme so it wouldn't necessarily be safe to just randomly rip them out. I'd have a bit more confidence if fluentui stops using them/deprecates them, since they're based off the fluentui Semantic colors.

Reply:
fluentui like the web one? That makes sense.

Reply:
Yeah, the web one. You can see those colors here: https://github.com/microsoft/fluentui/blob/dab45a2afe609709735b9b6e604a1af40d50e809/packages/theme/src/types/ISemanticColors.ts

Reply:
Thinking about this more, I think if we _did_ want to deprecate those colors (along with any others) in favor of our tokens, we would just have to rewrite the themes/components we export to not use those colors (default, win32, android, apple themes) and then mark the set as deprecated? Definitely out of scope though I'm just musing.

Reply:
Yeah I think that's how it would go.

Reply:
I am wondering if there's a better place to introduce the alias tokens than in the paletteFromFabricColors function considering this function is sort of on a deprecation path.  Let me know if you want a more specific suggestion and I'll look into it

Reply:
I can think of adding it to createOfficeTheme as a ThemeRecipe for the ThemeReference, but otherwise can't really think of anything better. If you have any better suggestions, let me know, but that's what I'll pivot to.

Reply:
I think this works well actually to set up for the "layering" or "recipe" part of putting the tokens together. The ThemeRecipe approach allows us to have information about the Office host available to determine alias theme colors.